generated/summaries/data_types.md

```diff
@@ -3,18 +3,102 @@ This is the course header. This will be added on top of every page. Do to [DoDAO
 
  ---
  
- ## Blockchain Basics
+ ## Types
  
  **Value Types**        
-Some Summary of Value Types   
-- Value type1
+The following types are called value types because variables of these types will always be passed by value, i.e. they are always copied when used in function arguments or asssignments.
```

Comment:
Also for the value types the value is stored in the variable and in reference types the reference or pointer is stored in the variable